ast/src/object/mod.rs

```diff
@@ -50,10 +50,28 @@ pub struct PILGraph {
 
 #[derive(Clone)]
 pub enum TypeOrExpression {
```

Comment:
TypeDeclarationOrExpression?

Reply:
In fact, this enum doesn't seem to be used anymore. It seems to have been abandoned after the recent refactor.